### PR TITLE
JavaScript capstone project

### PR DESCRIPTION
**Pull request by VTY1999**
## Changes made in this PR:
- Create the home page
- Set up the project with Webpack and Jest
- Display comments for a given item on the Comments pop-up
- Display Comments pop up with selected item's details
- Add comments counter
- Add footer
- Add header
- Display the number of likes for each item on the Homepage
- Display list of items on the Homepage
- Create feature: add a new comment
- Create feature: add new like
- Add all items counter on the Homepage
- Add tests for items counter
- Add tests for comments counter

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    submissionFail.innerHTML = 'Submission failed. Please try again.';
+  }
+});
+
+// mainContainer.addEventListener('click', (e) => {
+//   if (e.target.className === 'reserve-btn') {
+//     const id = e.target.getAttribute('data-target');
+//     displayData(id);
+//   }
+// });
```
Comment (eri8-9) [inline src/index.js:68]: - [ ] It would be great if we don't have commented code. 😉

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+{
+  fetchMovies,
+  userNameInput,
+  userCommentInput,
+  submissionFail,
+  postCommentsBtn,
+  mainContainer,
+  bigCommentsDiv,
+  detailsContainer,
+  // LogoContainer,
```
Comment (eri8-9) [inline src/index.js:17]: - [ ] It would be great if we don't have commented code. 😉

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import './style.css';
+import Icon from './images/icon.png';
+import rendering from './rendering.js';
+// import logoImage from './assets/Logo.png';
```
Comment (eri8-9) [inline src/index.js:4]: - [ ] It would be great if we don't have commented code. 😉

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+    <title>Capstone</title>
+</head>
+<body>
+    <header>
+      <div class="logo">
+        <!-- Logo Image added dynamically via JS -->
```
Comment (eri8-9) [inline src/index.html:13]: - [ ] It would be great if we don't have commented code. 😉

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+        <nav>
+          <ul class="navlist">
+            <li><a id="active" class="nav-items" href="">Movies</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <!-- Movies section generated dynamically -->
```
Comment (eri8-9) [inline src/index.html:28]: - [ ] It would be great if we don't have commented code. 😉

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️

- [x] The student recorded a video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - All the members in the video should appear in the video and talk about the project.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @VTY1999 and @SirriRyisa,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [x] **[REQUIRED]**  Good job with the video presentation. However, it is required that all the members of the team should appear in the video and talk about the project. The video should contain a highlight about a piece of the code the students find interesting or are proud of making.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VTY1999** [comment]
My partner said she won't be available today

**Comment by julie-ify** [comment]
Ok, Please make the other changes and submit okay.

**Review by FrederickMih — APPROVED** [review]
Hi @VTY1999, @SirriRyisa , 

### Congratulations!


In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**